docs/source/details/backendconfig.rst

```diff
@@ -74,6 +74,23 @@ Explanation of the single keys:
 Any setting specified under ``adios2.dataset`` is applicable globally as well as on a per-dataset level.
 Any setting under ``adios2.engine`` is applicable globally only.
 
+HDF5
+^^^^
+
+A full configuration of the HDF5 backend:
+
+.. literalinclude:: hdf5.json
```

Comment:
@franzpoeschel I just realized I forgot to add a `hdf5.json` file here :zany_face:

Reply:
Ah, I accidentally named it `json.json`

Reply:
-> #1169